ML/test.py

```diff
@@ -0,0 +1,46 @@
+import pandas as pd
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.preprocessing import StandardScaler
+from sklearn.cluster import KMeans
+import matplotlib.pyplot as plt
+from scipy.sparse import hstack
+from sklearn.manifold import TSNE
+
+# Step 1: Load the data from the CSV file
+df = pd.read_csv('ML/swe_train.csv')
```

Comment:
I think it would be good if there were a comment about which dataset this is.  Is this the same one that the notebook use? 

If so I think it would be good to name the files the same thing? 

Otherwise I think it would be good with a link making it easy to find the dataset

Reply:
This whole file was some initial testing, juts sandboxing really done by @WTANKW it has been left in here since we did some synchronization work with git... 

will remove this from the PR